image/templates/helm/stackrox-secured-cluster/templates/sensor.yaml.htpl

```diff
@@ -27,6 +27,7 @@ spec:
       annotations:
         traffic.sidecar.istio.io/excludeInboundPorts: "8443,9443"
         {{- include "srox.podAnnotations" (list . "deployment" "sensor") | nindent 8 }}
+        openshift.io/required-scc: stackrox-sensor
```

Comment:
Let me pick this example to highlight one thing. The `stackrox-sensor` SCC will be created only if we select a checkbox for operator installs (set a value on helm installs). In all other cases, sensor will run with the stock `restricted-v2` SCC.
Sensor does not need the custom SCC and there is no reason to use it. We should stick to `restricted-v2` wherever possible (this applies also to other services).